llvm/docs/LangRef.rst

```diff
@@ -16574,21 +16574,35 @@ type.
 
 Semantics:
 """"""""""
+Follows the semantics of minNum in IEEE-754-2008, except that -0.0 < +0.0 for the purposes
+of this intrinsic. As for signaling NaNs, per the minNum semantics, if either operand
```

Comment:
Describe with 2019 name. The 2008 version can be mentioned as a historical footnote